curriculum/js0/5.js

```diff
@@ -7,7 +7,12 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
+  if(a>b && a>c)
+    return a;
+  if(b>a && b>c)
+    return b;
+  if(c>a && c>b)
```

Comment:
don't need an if statement here. If it didn't return a or b, it must be c. so no need to for if statement.